lib/recurly/recurly_resource.php

```diff
@@ -116,9 +116,8 @@ function ($item) use ($setter) {
                                 if (property_exists($item, 'object')) {
                                     $item_class = static::resourceClass($item->object);
                                 } else {
-                                    // TODO: Ensure that there is a hintArrayType method
                                     $item_class = static::hintArrayType($setter);
-                                    if (!preg_match('/^Recurly/', $item_class)) {
+                                    if (!preg_match('/^\\\Recurly/', $item_class)) {
```

Comment:
So basically it's changing from `Recurly` to `\Recurly`?

Reply:
Yes, that's what is happening here. This bit of code has actually never been run before. It always should have been `\Recurly`